### PR TITLE
add kylin testnetwork

### PR DESCRIPTION
**Pull request by samelamin**
adding kylin testnet

### CODE REVIEW

File: packages/apps-config/src/endpoints/testing.ts [hunk 1]
```diff
@@ -192,6 +192,13 @@ export function createTesting (t: TFunction): LinkOption[] {
         'KILT Protocol': 'wss://full-nodes.kilt.io:9944/'
       }
     },
+    {
+      info: 'kylin',
+      text: t('testnet.kylin-node.co.uk', 'Kylin Testnet', { ns: 'apps-config' }),
+      providers: {
+        'Kylin Network': 'wss://testnet.kylin-node.co.uk'
+      }
+    },
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:201]: The same params are used here - https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/endpoints/testingRelayRococo.ts#L245-L252

Is this a parcahain on Rococo or a stand-alone network?

**Comment by samelamin** [comment]
This is meant to be a standalone network with a different url

**Comment by jacogr** [comment]
All ok, see the `rpc` vs `testnet` indicator now... (Although neither seems to load for me)

**Comment by samelamin** [comment]
Hi @jacogr Sorry I didnt really go into alot of detail on this one

What we are trying to do is deploy a secondary test network that our community can start running validators on. I assume this is the correct place to add it

I am not sure why the url arent resolving for you. But I redeployed both load balancers and tested both urls and seem to be fine

https://testnet.kylin-node.co.uk
https://rpc.kylin-node.co.uk

**Comment by samelamin** [comment]
Also another question for you Jacogr, I noticed that when I connect to my current node in production the css colour I set isn't taken affect 

Do I need to make CSS changes on my node, because I was under the impression the colours I set here should automatically take affect when connecting to the node via the Polkadot js app. 

Forgive me because I am not a front end developer but I don't see any obvious errors on the console that might indicate I did not set something up right

**Comment by samelamin** [comment]
I think I figured out the issue, it's my node that was named incorrectly! 

> Also another question for you Jacogr, I noticed that when I connect to ourcurrent node in production the css colour I set isn't taken affect
> 
> Do I need to make CSS changes on my node, because I was under the impression the colours I set here should automatically take affect when connecting to the node via the Polkadot js app.
> 
> Forgive me because I am not a front end developer but I don't see any obvious errors on the console that might indicate I did not set something up right
> 
> Could you help me understand where the styles are being set and if I'm lucky why ours isn't?
> 
> My suspicion is because I only added a logo and what I really need is an SVG file that describes the UI. Atleast that's what I can tell from the most recent pr to "fix colours" by another project

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**